pwnlib/tubes/ssh.py

```diff
@@ -1559,7 +1564,12 @@ def download_dir(self, remote=None, local=None):
 
             local_tar = tempfile.NamedTemporaryFile(suffix='.tar.gz')
             self.download_file(remote_tar, local_tar.name)
-
+            
+            #Delete temporary tarfile from remote host
+            if(self.sftp):
```

Comment:
Python here :) See [PEP-8](https://peps.python.org/pep-0008/)
```suggestion
            # Delete temporary tarfile from remote host
            if self.sftp:
```